GuessTheWord-Starter/app/src/main/java/com/example/android/guesstheword/screens/game/GameFragment.kt

```diff
@@ -49,46 +50,42 @@ class GameFragment : Fragment() {
 
         viewModel = ViewModelProvider(this).get(GameViewModel::class.java)
 
+        viewModel.score.observe(viewLifecycleOwner, { newScore ->
+            binding.scoreText.text = newScore.toString()
+        })
+
+        viewModel.word.observe(viewLifecycleOwner, { newWord ->
+            binding.wordText.text = newWord
+        })
+
+        viewModel.eventGameFinish.observe(viewLifecycleOwner, { hasFinished ->
+            if(hasFinished) gameFinished()
```

Comment:
Fix format of file please

Reply:
This is the way the code is formatted in the codelab, so I'm assuming it's the "Kotlin way". But I can format it some other way. Which changes do you suggest?